### PR TITLE
Added EXDATE field

### PR DESCRIPTION
**Pull request by LucaCoduriV**
I've added a field that indicates dates to exclude from rrule.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,57 +100,58 @@ END:VCALENDAR
...
-         "dtstart":"1997-07-14T17:00:00.000Z",
-         "dtend":"1997-07-15T03:59:59.000Z",
-         "summary":"Bastille Day Party",
-         "geo":{
-            "latitude":48.85299,
-            "longitude":2.36885
-         }
-      }
-   ]
+    "version": "2.0",
```
Comment (TesteurManiak) [inline README.md:103]: It seems like your formatter is configured differently, you shouldn't push files that do not need to be edited such as this one.

File: README.md [hunk 2]
```diff
@@ -100,57 +100,58 @@ END:VCALENDAR
...
+                "longitude": 2.36885
+            }
+        }
+    ]
 }
 ```
 
 ## Supported Properties
 
-* VERSION
```
Comment (TesteurManiak) [inline README.md:130]: Same as for the JSON snippet, it was uneeded to replace the `*` by a `-` and those editions should not have been pushed.

**Review by TesteurManiak — APPROVED** [review]
While editing the README was unneeded, the rest of the code is relevant, if the unit tests are still passing and there is no regression in the global code coverage this MR will be merged.

File: README.md (hunk 2 above)
Comment (LucaCoduriV) [reply README.md:130]: Yes sorry, it was because of my formatter

**Pull request merged**